### ISSUE TITLE
Play work but youtube search don't

### ISSUE THREAD

**Issue by hazzeee** [Splamy/TS3AudioBot#871]
Hi,

I have problem with my bots, the "!play" command work, but !search from youtube don't.
Search worked 2 days ago and stopped work.

When i try yts, it says:
 <11:42:20> "TS3AudioBot": Error: youtube-dl failed to load the resource.

But when i try same song with link and !play, it work.

How to fix this?

**Comment by Flakebi** [comment]
Can you try updating youtube-dl to the latest version here? https://yt-dl.org/
Everything works fine for me afterwards.

**Comment by farfik1** [comment]
> Can you try updating youtube-dl to the latest version here? https://yt-dl.org/
> Everything works fine for me afterwards.

Youtube-dl blocked by DMCA. That's why youtube-dl was giving a warning. I did this update but I still get the same warning.

**Comment by Flakebi** [comment]
I hope this issue has been solved. Beware that youtube changes quite often, so youtube-dl should be updated regularly.
If there are still problems, please reopen or create a new issue.

**Comment by lgund** [comment]
The problem is still there. I´ve the last youtube-dl working on my maschine but got always the same error. Sometimes I can spam the play command and after 2-3 times it work. I am not sure why he can´t load the resource. Has youtube-dl some log files?
